doc/sphinx/source/vp/filters.rst

```diff
@@ -0,0 +1,393 @@
+.. _filters:
+
+Filtering data
+==============
+
+Introduction
+------------
+
+In PDF fits, not all the data provided by the experimental
+collaborations are useful. For example, we may wish to discard certain
+datapoints for which we know small-x resummation or electroweak
+corrections are important. These effects are problematic since we know
+them to be important, but we cannot account for them.
+
+In this light, we produce cuts of the data, by filtering data points
+which we know are free of the above and other problems.
+
+In :ref:`validphys <vp-index>`, the cuts are handled by the :py:mod:`validphys.filters`
+alongside filter definitions and defaults found within
+:py:mod:`validphys.cuts`.
+
+Cuts as declarative filters
+---------------------------
+
+Due to the nature of data cuts, it is important to be transparent about
+which cuts are being applied to which dataset and/or process. Moreover,
+it is useful for the rules defining the data cut to be readable such
+that a non-developmental user can read and understand the nature of the
+rule by making these rules functions of kinematic variables such as
+``p_T`` or ``Q2``.
+
+In much the same vein, it is useful for any default values used in the
+rules to be readily accessible. For example, suppose there is a minimum
+value for the square transferred momenta in the DIS process ``q2min``,
+that is used widely by many different rules. It is important for this
+variable to be in an obvious and easily accessed location.
+
+Defaults
+--------
+
+There are certain values which are commonly used by many rules. For
+example, the value ``q2min`` usually takes the value ``3.49`` or
+``w2min`` is usually set to ``12.5``.
+
+It is thus useful to define these default values somewhere. These values
+can be found within ``validphys.cuts`` inside the ``defaults.yaml``
+file. One can overwrite these values and this is discussed later.
+
+Filters
+-------
+
+In ``validphys 2`` the default filter rules used can be found in the
+``validphys.cuts`` module within the ``filter.yaml`` file. This file is
```

Comment:
If you could add a link to the file in the repo it would be nice, so someone reading the docs can immediately check these out.